build/charts/antrea/values.yaml

```diff
@@ -67,6 +67,26 @@ ovs:
   # configuration).
   hwOffload: false
 
+secondaryNetwork:
+  ovs:
+    # -- Enable secondary network configuration in Antrea.
```

Comment:
Guess you should update this comment to: Enable OVS bridge configuration for secondary network.

Reply:
Aligned.